backEnd/src/themes/themes.service.ts

```diff
@@ -10,23 +10,64 @@ import { CreateThemeDto } from './dto/create-theme.dto';
 export class ThemesService {
   constructor(@Inject(PrismaService) readonly prisma: PrismaService) {}
 
-  findAll() {
-    return this.prisma.theme.findMany();
+  async findAll() {
+    return await this.prisma.theme.findMany();
   }
 
-  create(createThemeDto: CreateThemeDto) {
-    const theme = this.prisma.theme.findUnique({
+  async create(createThemeDto: CreateThemeDto) {    
+    const theme = await this.prisma.theme.findUnique({
       where: {
         name: createThemeDto.name,
       },
-    });
+    });    
 
     if (theme) {
       throw new UnprocessableEntityException('Theme already exists.');
     }
 
-    return this.prisma.theme.create({
+    return await this.prisma.theme.create({
```

Comment:
Good catch!

Here we don´t need to await the promise as we don't have a try/catch and we are returning it
```suggestion
    return this.prisma.theme.create({
```